dll/win32/atl80/atl80.manifest

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
-  <assemblyIdentity type="win32" name="Microsoft.VC80.ATL" version="8.0.50727.4053" processorArchitecture="" publicKeyToken="1fc8b3b9a1e18e3b"/>
+  <assemblyIdentity type="win32" name="Microsoft.VC80.ATL" version="8.0.50727.4053" processorArchitecture="*" publicKeyToken="1fc8b3b9a1e18e3b"/>
```

Comment:
There are 5 other occurrences (2 gdiplus and 3 msxml), in case they would apply too:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=processorArchitecture%3D%22%22

Reply:
Done